modules/ROOT/pages/dataweave-cookbook-defaults.adoc

```diff
@@ -65,23 +69,45 @@ and
 xref:dataweave-flow-control.adoc#control_flow_else_if[else-if]
 statements to specify a default value.
```

Comment:
"Another method for providing a default value is using xref:dataweave-flow-control.adoc#control_flow_if_else[if-else] and xref:dataweave-flow-control.adoc#control_flow_else_if[else-if]. 

Under typical circumstances, providing a value ...results in expected output:
<example>

But if no value is specified, having a default configured results in usable output rather than <what>:"